src/search.ts

```diff
@@ -107,29 +101,26 @@ function getRequiredModules(requireNodes: Node[]): SearchValue {
 }
 
 /**
- * Returns list of positions where there are dynamic require calls
+ * Returns list of positions where there the require identifier is used other
+ * than in a call expression
  *
  * @param tree abstract syntax tree
  */
 function getDynamicRequireCalls(tree: Node): Position[] {
   const dynamicRequireCalls: Position[] = [];
-  walk.ancestor(tree, {
-
-    Identifier(n: Node, ancestors: Node[]) {
-      if (n.type === 'Identifier' && n && n !== undefined) {
-        if (n.name === 'require') {
-          // last element is current node and second to last it the node's
-          // parent
-          const parent: Node = ancestors[ancestors.length - 2];
-          if (parent.type !== 'CallExpression' ||
-              parent.arguments.length !== 1) {
-            // Dynamic Require call
+  walk.fullAncestor(tree, (n: Node, ancestors: Node[]) => {
```

Comment:
Why did this change to `fullAncestor`? What is the difference?

Reply:
According to the documentation, ```walk.ancestor``` only look at nodes that are expressions or statements. I found that my function didn't detect module.require calls because in this case, require is a property and not an expression or statement. ```walk.fullAncestor``` is supposed to check all nodes but it still did not work for module.require. I forget to change it back, but the new implementation gets the same result... do you think I should still change it back?

Reply:
If they do the same thing, it's probably OK to have both. We can change it back later if it doesn't work for some reason.